### PR TITLE
Issue#32

### PR DESCRIPTION
**Pull request by Mellorison**
Added Placeholder Gallery under the "EDI Insights" Patient Dashboard for the demo while I fix the bug preventing the official gallery load process.

### CODE REVIEW

**Comment by hallix** [comment]
@kaykabwe @csamiselo Help urgently wanted to review this pull request, it will be part of the demo build.

**Comment by kaykabwe** [comment]
All good

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -13,13 +13,13 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".BootstrapActivity">
```
Comment (hallix) [inline app/src/main/AndroidManifest.xml]: @Mellorison Revert login screen, currently skipping.

File: core/src/main/java/zm/gov/moh/core/service/SyncService.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
 public abstract class SyncService extends IntentService implements InjectableViewModel {
 
     protected Repository repository;
-    protected String accesstoken = "";
+    protected String accesstoken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyIjp7InV1aWQiOiJhM2FjYTE4MS1kMDJhLTRiODgtOTc1NC1lYWM0NWQzZGUzZmUiLCJkaXNwbGF5IjoiYW50aG9ueSIsInVzZXJuYW1lIjoiYW50aG9ueSIsInN5c3RlbUlkIjoiMy00In0sImlhdCI6MTU0MjE0MzU3NiwiZXhwIjoxNTkyMTQzNTc2fQ.DsDbPXwaZ5sg2SFCq1CBykITJjog-9u-4XzNGw9IYV8";
```
Comment (hallix) [inline core/src/main/java/zm/gov/moh/core/service/SyncService.java]: @Mellorison Remove access token

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (Mellorison) [reply app/src/main/AndroidManifest.xml]: Reverted.

**Pull request merged**